compute/gpu/how-to/use-scratch-storage-h100-instances.mdx

```diff
@@ -0,0 +1,62 @@
+---
+meta:
+  title: How to use the scratch storage on H100 GPU Instances
+  description: This page explains how to use the scratch storage on H100 GPU Instances
+content:
+  h1: How to use the the scratch storage on H100 GPU Instances
+  paragraph: This page explains how to use the scratch storage on H100 GPU Instances
+tags: H100 gpu scratch-storage scratch storage
+dates:
+  validation: 2023-09-18
+  posted: 2023-09-18
+categories:
+  - compute
+---
+
+Scaleway H100 GPU Instances are equipped with additional scratch storage. This form of temporary local storage operates differently from our regular local storage.
+
+Scratch storage temporarily accommodates data during computational or data processing tasks. It finds common use for storing intermediate results, processing input data, or holding output data before it is moved to more permanent storage.
+
+Unlike conventional storage, scratch storage lacks features like snapshots, backups, or restores. Furthermore, it isn't designed for downloading images; its main function is to serve as a cache. Typically, data in scratch storage is deleted once the computation or processing task concludes.
+
+Scaleway's H100 GPU Instances use NVMe disks for their scratch storage, which are fine-tuned for high-speed data access. This design ensures fast read and write speeds, which is crucial for applications demanding extensive data processing capabilities.
+
+<Message type="note">
+  Scratch storage does not survive stopping the server, doing a full stop/start cycle will erase the scratch data. However, doing a simple reboot or using the stop in place function will keep the data.
+</Message>
+
+## What can I use scratch storage for?
+
+For example, the GPU Render-S requires several minutes (typically ranging from 6 to 12 minutes) to fetch a 400GB image, a delay that impedes the user's ability to promptly initiate model training. This lag becomes particularly problematic in the realm of AI, where a 400GB dataset is considered relatively small.
+To address this issue, we have implemented scratch storage. Unlike traditional storage, scratch storage does not necessitate the downloading of the entire image (and therefore cannot be used for backup restoration), but it boasts the capability to feed data into the GPU at a significantly accelerated rate. This enhancement allows us to provide the GPU with a substantial amount of scratch storage, ensuring a swifter and more efficient data input process.
+
+<Message type="note">
+      The maximum possible size for scratch storage is
+      * for H100-1-80G Instances: 1920GB
+      * for H100-2-80G Instances: 3940GB
+</Message>
+
+## How to use scratch storage using Scaleway CLI and console
```

Comment:
```suggestion
## How to use scratch storage using the Scaleway CLI and console?
```

Reply:
@nerda-codes I think it was @RoRoJ who said it sounds weird in English to use a question mark after a `How to` ?
Just to check :)